formatTest/unit_tests/expected_output/variants.re

```diff
@@ -514,3 +514,25 @@ module Graph = {
 
 type Graph.node +=
   | Str = Graph.Str;
+
+type water = ..;
+
+type water +=
+  pri
+  | Ocean;
+
+type water +=
+  pri
```

Comment:
I like how we always break the lines here so that it's more obvious that the pri is used to decorate the entire list of constructors, as opposed to a single constructor.